sdcm/utils/latency.py

```diff
@@ -47,10 +49,15 @@ def collect_latency(monitor_node, start, end, load_type, cluster, nodes_list):
                     lat_value = max([float(val) for val in sequence])
             else:
                 lat_value = avg(sequence)
+                max_lat_value = max(sequence)
```

Comment:
Avi didn't want to use max.
He wanted to choose something that better demonstrate the progress.